lib/mix/tasks/server.ex

```diff
@@ -22,7 +22,7 @@ defmodule Mix.Tasks.Server do
       opts = Keyword.update!(opts, :port, &binary_to_integer(&1))
     end
 
-    opts = add_config(opts)
+    opts = add_config opts
```

Comment:
Was there a particular reason for this particular change?  It's not necessarily harmful, but it is inconsistent with the surrounding code, which might be a little weird readability-wise.